packages/frontend/src/components/common/Table/Table.styles.ts

```diff
@@ -2,7 +2,8 @@ import { Colors, HTMLTable } from '@blueprintjs/core';
 import { transparentize } from 'polished';
 import styled, { css } from 'styled-components';
 
-export const TABLE_HEADER_BG = Colors.LIGHT_GRAY4;
+// This color is is the default Mantine gray[0]
+export const TABLE_HEADER_BG = '#f8f9fa';
```

Comment:
I don't love this hardcoded color, but I decided it's ok because there are other color strings in this file. I couldn't find a reasonable way to bring the Mantine theme into this file.